src/core/annotation.js

```diff
@@ -619,8 +621,7 @@ var WidgetAnnotation = (function WidgetAnnotationClosure() {
     var data = this.data;
 
     data.annotationType = AnnotationType.WIDGET;
-    data.fieldValue = stringToPDFString(
-      Util.getInheritableProperty(dict, 'V') || '');
+    data.fieldValue = Util.getInheritableProperty(dict, 'V');
```

Comment:
Since the `V` entry can be an array, you probably need to modify `Util.getInheritableProperty` to handle that. I can see two possible solutions:
 - Modify `Util.getInheritableProperty` to use always use `Dict_getArray` instead of `Dict_get`. Even though it only looks like this utility function is used from `src/core/annotation.js`, I'm not sure that this is such a great solution!
 - Add a parameter to `Util.getInheritableProperty` that controls whether `Dict_getArray` is used, e.g. something like this:
```diff
diff --git a/src/core/annotation.js b/src/core/annotation.js
index 13aa204..d7ef2ca 100644
--- a/src/core/annotation.js
+++ b/src/core/annotation.js
@@ -619,8 +619,8 @@ var WidgetAnnotation = (function WidgetAnnotationClosure() {
     var data = this.data;
 
     data.annotationType = AnnotationType.WIDGET;
-    data.fieldValue = stringToPDFString(
-      Util.getInheritableProperty(dict, 'V') || '');
+    data.fieldValue = Util.getInheritableProperty(dict, 'V',
+                                                  /* getArray = */ true);
     data.alternativeText = stringToPDFString(dict.get('TU') || '');
     data.defaultAppearance = Util.getInheritableProperty(dict, 'DA') || '';
     var fieldType = Util.getInheritableProperty(dict, 'FT');
diff --git a/src/shared/util.js b/src/shared/util.js
index 48876da..614e7a0 100644
--- a/src/shared/util.js
+++ b/src/shared/util.js
@@ -874,9 +874,10 @@ var Util = (function UtilClosure() {
   };
 
   Util.getInheritableProperty = function Util_getInheritableProperty(dict,
-                                                                     name) {
+                                                                     name,
+                                                                     getArray) {
     while (dict && !dict.has(name)) {
-      dict = dict.get('Parent');
+      dict = getArray ? dict.getArray('Parent') : dict.get('Parent');
     }
     if (!dict) {
       return null;

```